data/json/items/armor/torso_clothes.json

```diff
@@ -1689,7 +1689,7 @@
       },
       {
         "id": "start_a_cult_tshirt",
-        "name": { "str": "activities for children t-shirt", "str_pl": "activities for children t-shirts" },
+        "name": { "str": "activities for children t-shirt" },
```

Comment:
The plural is here to suppress the warning about potential irregular plural form because the name contains `for`.